x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/support/DelegatedAuthorizationSupport.java

```diff
@@ -166,4 +166,12 @@ private Realm findRealm(String name, Iterable<? extends Realm> allRealms) {
         throw new IllegalArgumentException("configured authorization realm [" + name + "] does not exist (or is not enabled)");
     }
 
+    @Override
+    public String toString() {
+        return "security-delegated-authorization: "
+            + Security.DELEGATED_AUTHORIZATION_FEATURE.check(licenseState)
```

Comment:
```suggestion
            + "available: " 
            + Security.DELEGATED_AUTHORIZATION_FEATURE.checkWithoutTracking(licenseState)
```

I don't think we can just print out a `true`/`false` without an explanation of what it means.

But more significantly, calling `check` triggers license feature tracking. You have to be careful not to call `check` unless you know that a licensed feature is actually being used. In this case you don't because the lookup might be empty.

Reply:
Resolved. I switched to checkWithoutTracking.